src/DataTable.js

```diff
@@ -22,18 +21,36 @@ export function DataTable(props) {
     dataSource,
     refCallback,
     renderRow,
+    scrollable,
+    ListViewComponent,
+    userRealm,
     ...listViewProps,
   } = props;
+
+  const renderListView = () => (
+    <ListViewComponent
+      {...listViewProps}
+      ref={refCallback}
+      style={[defaultStyles.listview, listViewStyle]}
+      dataSource={dataSource}
+      renderRow={renderRow}
+    />
+  );
   return (
-    <View style={[defaultStyles.verticalContainer, style]}>
-      {typeof renderHeader === 'function' && renderHeader()}
-      <ListView
-        {...listViewProps}
-        ref={refCallback}
-        style={[defaultStyles.listview, listViewStyle]}
-        dataSource={dataSource}
-        renderRow={renderRow}
-      />
+    <View>
+      {
+        scrollable ?
```

Comment:
Probably add as false to defaultProps, rather than letting undefined float through (although undefined is falsy anyway)